src/test/js/reporters/accessibility-reporter/Report.html

```diff
@@ -0,0 +1,210 @@
+<head>
+  <meta charset="utf-8">
+  <!-- <meta http-equiv="Content-Security-Policy"
+      content="default-src *; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-inline' 'unsafe-eval'
+       https://cdnjs.cloudflare.com https://ajax.googleapis.com http://www.google.com"> -->
+  <script src="resources/angular.min.js"></script>
+
+  <title>IDAM Web Accesibility Test Report</title>
+  <meta name="title" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="resources/css/all.css">
+  <style>
+    pre {
+      white-space: pre-wrap;       /* css-3 */
+      white-space: -moz-pre-wrap;  /* Mozilla, since 1999 */
+      white-space: -pre-wrap;      /* Opera 4-6 */
+      white-space: -o-pre-wrap;    /* Opera 7 */
+      word-wrap: break-word;       /* Internet Explorer 5.5+ */
+    }
+    body {
+      background-color: lightgray;
+    }
+    i{
+      margin:10px
+    }
+    .stats{
+      font-size: 24px;
+    }
+
+    .header{
+      background-color: darkslategrey ;
+      border-bottom-style: solid;
+      border-bottom-width: 5px;
+      border-color: #ff0000;
+      margin-bottom: 5px;
+      color: white;
+    }
+
+    #reportHeaderTitle{
+      font-size: 24px;
+    }
+    #reportHeaderAppName{
+      font-size: 16px;
+
+    }
+    .header .titleText {
+      display: inline-block;
+      width: 60%;
+      text-align: center;
+    }
+
+    .header .titleText>div {
+      display: inline-block;
+      /* width: 40%; */
+      text-align: center;
+    }
+
+    .header .stats {
+      display: inline-block;
+      width: 30%;
+    }
+
+    .testResult{
+      border-top-style: groove;
+      border-top-width:5px ;
+      background-color: white;
+      margin: 0% 15%;
+    }
+
+
+    .testTitle{
+      font-size: 24px;
+      font-weight: bold;
+      padding: 5px;
+    }
+    .testTitle.pass{
+      color: green;
+    }
+    .testTitle.fail{
+      color: red;
+    }
+
+    .a11yResult.show{
+      display: block;
+    }
+
+    .resultTable,.a11yResult h4{
+      margin-left: 2%;
+    }
+
+    .issuesTable{
+      margin-left: 4%;
+      border-top-width: 5px;
+      border-top-color: red;
+      border-top-style: solid;
+      margin-bottom: 2px;
+
+    }
+
+    .a11yResult.hide{
+      display: none;
+    }
+
+    table{
+      border-collapse: collapse;
+      margin: 2px;
+    }
+    .a11yResult table tr th,.a11yResult table tr td{
+      padding: 2px;
+      text-align: left;
+      border-style:groove
+    }
+    .a11yResult table tr th{
+      background-color: lightgray;
+    }
+
+
+
+  </style>
+</head>
+<body>
+<div ng-app="myApp" ng-controller="myCtrl">
+  <div class="header">
+    <div class="titleText">
+      <div>
+        <i class="fas fa-universal-access" style="font-size:60px;color:white;"> </i>
+      </div>
+
+      <div>
+        <div id="reportHeaderTitle">Accessibility Test Report for Expert UI</div>
```

Comment:
Expert UI to be changed to **IDAM Web Public UI**.

Reply:
This has been updated now. Thank you!